charts/osm/crds/policy.yaml

```diff
@@ -85,6 +86,7 @@ spec:
                       number:
                         description: Port number of this port.
                         type: integer
+                        pattern: ^[1-9]\d*$
```

Comment:
would be good to prevent non-uint16 ranges. guess this could also be done by validating webhook

Reply:
Done!